modules/Microsoft.Synapse/workspaces/keys/deploy.bicep

```diff
@@ -0,0 +1,56 @@
+@description('Required. Encryption key name.')
+param name string
+
+@description('Required. Synapse workspace name.')
+param workspaceName string
+
+@description('Optional. The geo-location where the resource lives.')
```

Comment:
Is location required for the child? If not please remove. If it is please add that to the outputs

Reply:
I've seen other cases in which the location is not a child property and the telemetry uniquename is generated without it, only using the deployment name. Wouldn't it be the case to align with the same approach here?

Reply:
Created #1704

Reply:
Sounds good 👍 
Shouldn't the static validation fail in this cases if location is not returned as an output?

Reply:
Actually no. The test is smart enough to only test for it if a resource inside the template has a location property. And only then it is actually a useful output :)